src/main/java/org/folio/helper/ExpectHelper.java

```diff
@@ -52,10 +52,20 @@ private Future<ReceivingResults> processExpectPieces(ExpectCollection expectColl
       .map(this::updatePieceRecords)
       // 3. Update received piece records in the storage
       .compose(piecesGroupedByPoLine -> storeUpdatedPieceRecords(piecesGroupedByPoLine, requestContext))
-      // 4. Return results to the client
+      // 4. Update PO Line status
+      .compose(piecesByPoLineIds -> updateOrderAndPoLinesStatus(piecesByPoLineIds, requestContext))
+      // 5. Return results to the client
       .map(piecesGroupedByPoLine -> prepareResponseBody(expectCollection, piecesGroupedByPoLine));
   }
 
+  private Future<Map<String, List<Piece>>> updateOrderAndPoLinesStatus(Map<String, List<Piece>> piecesGroupedByPoLine, RequestContext requestContext) {
+    return updateOrderAndPoLinesStatus(
+      piecesGroupedByPoLine,
+      requestContext,
+      poLines -> {}
+    );
+  }
```

Comment:
need to add module permission to expect helper now(befo this change we had only for checking and receive), the same that I did in this PR: https://github.com/folio-org/mod-orders/pull/861

Reply:
"finance.funds.item.get", and this one also should be added after https://github.com/folio-org/mod-orders/pull/855

Reply:
I see that we not invoking logic ti updateOrderStatus here, so this finance permission is not needed